operator/rpc.go

```diff
@@ -77,6 +137,10 @@ func (r *RPCServer) connectionHandler(out http.ResponseWriter, in *http.Request)
 	}
 
 	peer := &Peer{conn: conn, op: r.op}
+	if err := peer.sendJSON(r.server.clientConfig); err != nil {
+		r.Log().WithError(err).Error("Pushing ClientConfig:", err)
```

Comment:
You are logging the error twice. Just looked at it because i have to rebase onto it.

Reply:
Good point, we also did the same mistake five lines above :laughing: